community/csv/src/main/java/org/neo4j/csv/reader/BufferedCharSeeker.java

```diff
@@ -160,6 +145,11 @@ else if ( nextCh != untilChar && !isNewLine( nextCh ) && nextCh != EOF_CHAR )
                 }
                 else if ( isNewLine( ch ) )
                 {   // Found a new line, just keep going
```

Comment:
(At least the "just keep going" part of) the comment is now more confusing than helping to a new reader of the code. Btw, if my perfectionist side takes over and I go/went from helpful to annoying, please let me know ;-)